src/components/endpoints/Assets.vue

```diff
@@ -95,6 +95,7 @@ const create = async () => {
           controllerAddress: erc725AccountAddress,
           name: token.value.name,
           symbol: token.value.symbol,
+          creators: [erc725AccountAddress],
```

Comment:
Should we add the same to `deployLSP7DigitalAsset`? It has the same property as both functions' arguments share the same type (except that LSP7 just extends it).

Reply:
We can. Wasn't so crucial as we don't display this information in wallet, creators are only shown for LSP8.